Plugins/Public/base_plugin/Main.cpp

```diff
@@ -156,6 +156,8 @@ float siege_mode_damage_trigger_level = 8000000;
 //the distance between bases to share siege mod activation
 float siege_mode_chain_reaction_trigger_distance = 8000;
 
+set<uint> customSolarList;
```

Comment:
Is there a particular reason why you're not using an `unordered_set` here? Do the custom solars have to be sorted?